packages/rich-text-types/src/nodeTypes.ts

```diff
@@ -157,38 +157,26 @@ export interface EntryHyperlink extends Inline {
 }
 
 export interface TableCell extends Block {
-  nodeType: BLOCKS.TABLE_CELL;
+  nodeType: BLOCKS.TABLE_HEADER_CELL | BLOCKS.TABLE_CELL;
```

Comment:
Should we include `BLOCKS.TABLE_HEADER_CELL` since we already have `TableHeaderCell` that overrides this value?

Reply:
It's tricky, we have to allow both types in the general TableCell, so the schema validation allows both cells when validating a `TableRow` node